### PR TITLE
set location for client CA cert file

### PR DESCRIPTION
**Pull request by persiaAziz**

### CODE REVIEW

**Comment by scw00** [comment]
[approve ci autest]

**Comment by persiaAziz** [comment]
[approve ci autest]

File: iocore/net/SSLNetVConnection.cc [hunk 1]
```diff
@@ -980,6 +980,14 @@ SSLNetVConnection::sslStartHandShake(int event, int &err)
       } else {
         clientCTX = params->client_ctx;
       }
+
+      if (this->options.clientVerificationFlag && params->clientCACertFilename != nullptr && params->clientCACertPath != nullptr) {
```
Comment (zwoop) [inline iocore/net/SSLNetVConnection.cc]: Nit pick, but for future, I'd much prefer e.g.

```
this->options.clientVerificationFlag && (params->clientCACertFilename != nullptr)
```

it just looks better, and feels a lot safer. Yes, I know they are the same.

**Comment by zwoop** [comment]
Cherry-picked to 7.1.x

**Pull request merged**